### ISSUE TITLE
close earthbar tabs when clicking outside of them

### ISSUE THREAD

**Issue by ikethefifth** [sgwilym/twodays-crossing#5]
I would like the earthbar tabs to close when I click anywhere outside of them instead of just on their name.

**Comment by judytuna** [comment]
I would, too! However, is this a task for earthbar? https://github.com/earthstar-project/react-earthstar/tree/master/src/components/earthbar

**Comment by ikethefifth** [comment]
> I would, too! However, is this a task for earthbar? https://github.com/earthstar-project/react-earthstar/tree/master/src/components/earthbar

Yes! I made this before I understood how the earthbar worked. I'll add it as an issue over there and close it here. Thanks!

**Comment by sgwilym** [comment]
I think this might be better here, actually: https://github.com/earthstar-project/react-earthstar/issues/36

**Comment by judytuna** [comment]
Sounds good! Thank you =)
